test/models/trip_test.dart

```diff
@@ -0,0 +1,251 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/rides/models/ride.dart';
+import 'package:motis_mitfahr_app/util/trip/trip.dart';
+
+import '../util/factories/drive_factory.dart';
+import '../util/factories/ride_factory.dart';
+import '../util/mock_server.dart';
+import '../util/mock_server.mocks.dart';
+
+void main() {
+  final MockUrlProcessor tripProcessor = MockUrlProcessor();
+  //setup muss in jeder Testklasse einmal aufgerufen werden
+  setUp(() async {
+    MockServer.setProcessor(tripProcessor);
+  });
+  group('Trip.duration', () {
+    test('returns the duration of a ride', () {
```

Comment:
okay but now can you write "trip" everywhere instead of ride/drive

Reply:
I renamed the variables. But I would leave the description cause they show that we cover rides and drives

Reply:
I meant in the group and test description